test/unit/services/DateService.spec.ts

```diff
@@ -0,0 +1,132 @@
+import { DateService, TimespanResolution } from "services/DateService";
+
+
+type TimeArray = [
+  days: number,
+  hours: number,
+  minutes: number,
+  seconds: number
+];
+type TestData = [
+  resolution: TimespanResolution,
+  timeArray: TimeArray,
+  expected: string
+];
+
+describe("DateService", () => {
+  let dateService: DateService;
+  beforeAll(() => {
+    dateService = new DateService();
+  });
+
+  describe("#ticksToTimeSpanString", () => {
+    /* prettier-ignore */
+    const testData: TestData[] = [
+      [TimespanResolution.largest, [1, 1, 1, 1], "1 day, 1 hour"],
+      [TimespanResolution.largest, [1, 1, 1, 0], "1 day, 1 hour"],
+      [TimespanResolution.largest, [1, 1, 0, 1], "1 day, 1 hour"],
+      [TimespanResolution.largest, [1, 1, 0, 0], "1 day, 1 hour"],
+      [TimespanResolution.largest, [1, 0, 1, 1], "1 day, 1 minute"],
+      [TimespanResolution.largest, [1, 0, 1, 0], "1 day, 1 minute"],
+      [TimespanResolution.largest, [1, 0, 0, 1], "1 day, 1 second"],
+      [TimespanResolution.largest, [1, 0, 0, 0], "1 day, 0 seconds"],
```

Comment:
@dkent600 
To go with the "quickest" solution only this case is a bit weird.
But as discussed. neglectible